Wrong binaries in prebuild versions for electron v70/v73

It seems that binaries that are labeled by `zeromq-v5.1.0-electron-v70-X` and `zeromq-v5.1.0-electron-v73-X` are actually built against NODE_MODULE_VERSION  47 and 48 respectively. 

```
Error: The module `...\node_modules\zeromq\build\Release\zmq.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 47. This version of Node.js requires
NODE_MODULE_VERSION 70. Please try re-compiling or re-installing

```

```
Error: The module `...\node_modules\zeromq\build\Release\zmq.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 48. This version of Node.js requires
NODE_MODULE_VERSION 70. Please try re-compiling or re-installing

```

@andsav Attempting to run this command produces an error for me:

Command: `node-gyp rebuild --target=6.0.0 --arch=x64 --dist-url=https://electronjs.org/headers`

Error:
`LINK : fatal error LNK1104: cannot open file 'C:\Users\...\node_modules\zeromq\build\Release\..\..\windows\lib\libzmq.lib' [C:\Users\...\node_modules\zeromq\build\zmq.vcxproj]`

This is using Visual Studio 2017 on Windows 10.

--- a/package.json
+++ b/package.json
@@ -9,15 +9,15 @@
     "url": "https://github.com/zeromq/zeromq.js.git"
   },
   "dependencies": {
-    "nan": "^2.10.0",
-    "prebuild-install": "5.2.1"
+    "nan": "^2.14.0",
+    "prebuild-install": "^5.3.2"
   },
   "devDependencies": {
     "electron-mocha": "^6.0.0",
     "jsdoc": "^3.5.4",
     "mocha": "^5.0.0",
     "nyc": "^12.0.2",
-    "prebuild": "^8.1.0",
+    "prebuild": "^9.1.1",
     "semver": "^5.4.1",
     "should": "^13.0.0"
   },

